examples/README.md

```diff
@@ -1,64 +1,29 @@
-gRPC in 3 minutes (Go)
-======================
+# gRPC Hello World
 
-BACKGROUND
--------------
-For this sample, we've already generated the server and client stubs from [helloworld.proto](helloworld/helloworld/helloworld.proto).
+Follow these setup to run the [quick start][] example:
 
-PREREQUISITES
--------------
+ 1. Get the code:
 
-- This requires Go 1.9 or later
-- Requires that [GOPATH is set](https://golang.org/doc/code.html#GOPATH)
+    ```console
+    $ go get -u google.golang.org/grpc/examples/helloworld/greeter_client
+    $ go get -u google.golang.org/grpc/examples/helloworld/greeter_server
```

Comment:
I don't think `-u` is recommended anymore, with modules it has a different behavior than before.

Reply:
Done: dropped `-u` flags.